extension/framework/framework.go

```diff
@@ -33,9 +35,28 @@ var log = logging.MustGetLogger("extest")
 func TestExtensions(c *cli.Context) {
 	buflog.SetUpDefaultLogging()
 
+	var config *util.Config
```

Comment:
Why not just use `config := util.GetConfig()`? As far as I understand, this can't fail.